src/_P076_HLW8012.ino

```diff
@@ -635,4 +738,18 @@ void IRAM_ATTR p076_hlw8012_cf_interrupt() {
   }
 }
 
+const __FlashStringHelper* p076_getQueryString(uint8_t query) {
+  switch (query)
+  {
+    case 0: return F("Voltage_V");
+    case 1: return F("Current_A");
+    case 2: return F("Active Power_W");
+    case 3: return F("Reactive Power_VAR");
+    case 4: return F("Apparent Power_VA");
+    case 5: return F("Power_Factor_cosphi");
+    case 6: return F("Energy_Ws");
```

Comment:
These strings are used as value names, but the rules parser doesn't handle/accept spaces in value names, so the remaining spaces should probably also be replaced by underscores.
And I'd expect the previous value names `Voltage`, `Current`, `Power` and `PowerFactor`, to be available, and preferably as the defaults (would require swapping case 3 and case 5, and also some other code adjustments), to not break existing installs and rules based on that.